squabble/config.py

```diff
@@ -238,26 +252,27 @@ def _extract_file_rules(text):
     rules = {
         'enable': {},
         'disable': [],
-        'skip_file': False
+        'skip_file': False,
+        'skip_lines': []
     }
 
-    comment_re = re.compile(
-        r'--\s*'
-        r'(?:squabble-)?(enable|disable)'
-        r'(?::\s*(\w+)(.*?))?'
-        r'$', re.I)
-
-    for line in text.splitlines():
+    for line_num, line in enumerate(text.splitlines()):
         line = line.strip()
-
-        m = re.match(comment_re, line)
+        m = re.match(COMMENT_RE, line)
         if m is None:
             continue
 
-        action, rule, opts = m.groups()
+        action, ctx, rule, opts = m.groups()
+        line_specific = ctx is not None
 
         if action == 'disable' and not rule:
-            rules['skip_file'] = True
+            if not line_specific:
+                rules['skip_file'] = True
+            else:
+                # Disregard specific rule for now
+                # Starts with 0, +1 additional line because
+                # we want to skip the *next* line
+                rules['skip_lines'].append(line_num + 2)
```

Comment:
This is *definitely* bikeshedding, and I know you're just extending what I had for disabling files, but maybe this should be called `disabled_lines` to align with the flag that's used in the comment?